spec/outputs/elasticsearch_spec.rb

```diff
@@ -597,6 +612,12 @@ def mock_actions_with_response(*resp)
         end
 
         it "successful requests should not be appended to retry queue" do
+          # insist indices are empty
+          Stud::try(10.times) do
+            r = @es.search
+            insist { r["hits"]["total"] } == 0
```

Comment:
I agree with you, but, I would like to have it be consistent across the file for now. once we fix the flakiness. I will go ahead and make PR to `s/insist/expect`